tests/vmi_configuration_test.go

```diff
@@ -898,7 +902,9 @@ var _ = Describe("[sig-compute]Configurations", func() {
 				By("Starting a VirtualMachineInstance")
 				// Retrying up to 5 sec, then if you still succeeds in VMI creation, things must be going wrong.
 				Eventually(func() error {
-					vmi = tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
+					vmi = libvmi.NewAlpine()
+					// Namespace must be specified for the admission webhook
```

Comment:
To clarify, I am fine with this comment. Maybe it is worth ref the issue here, but maybe it is controversial (if it is really a bug or not) so I do not want to insist on this.

Reply:
If it's not set, the "feature" doesn't work, and the VM is either allowed to be created or doesn't have the resources scaled.
I think it's a bug too and this way, indeed it is at least exposed, instead of being hidden in by the utils function.

Reply:
The `LimitRange` is not a KubeVirt feature though: https://kubernetes.io/docs/concepts/policy/limit-range/

Since the limits are evaluated at admission, it may be limitation of the webhooks, as the namespace is not known at that time.

Reply:
The bug as I see it, is that got implemented in the webhook and not in the controller.

Reply:
Kubevirt has its own LimitRange implementation. Are you talking about this one?

Reply:
These tests show that the webhook assumed the namespace is specified in the spec, which is wrong.
Well, unless the client is responsible to move the namespace specified in the "Create" to the spec before it calls the API server to post the request. I am unsure what happens here, but requiring to set a namespace in the spec seems wrong to me.

Reply:
I have opened an issue to track this, should I link the issue in the test? https://github.com/kubevirt/kubevirt/issues/7681